dspace-spring-rest/src/main/java/org/dspace/app/rest/model/DirectlyAddressableRestModel.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * A directly addressable REST resource
+ * 
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ *
+ */
+public interface DirectlyAddressableRestModel extends RestModel {
```

Comment:
Here's the actual `DirectlyAddressableRestModel` class. I'm not sure I understand the terminology here, and why it is now "directly addressable" because it has a Category and Controller?  Maybe I'm the only one though who feels the name here is obscure/unclear?

Reply:
Maybe RestAddressableModel is the name appropriate for this interface.